packages/graphback-cli/src/templates/configTemplates.ts

```diff
@@ -99,13 +99,6 @@ export const createConfig = async (database: string, client: boolean) => {
     }
   };
 
-  if(database === "pg") {
```

Comment:
@wtrocki I have removed this from generation as `graphback config` is for people with existing systems.

Reply:
I could not find information how to add this so If we remove it here we need extra chapter in https://graphback.dev/docs/intro/existingproject

Reply:
It is there in **Next Steps**:

> 4. If you are using a PostgreSQL datasource you can migrate your database to match your schema by running graphback db. See Database Migrations for more.

Reply:
Right I could not find exacy format of the dbmigrations anywhere - we removed it from config and it is not documented in actual migrations doc